airbyte-ci/connectors/metadata_service/orchestrator/orchestrator/assets/connector_metrics.py

```diff
@@ -0,0 +1,62 @@
+#
+# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+#
+
+import json
+from typing import Optional
+
+import sentry_sdk
+from dagster import OpExecutionContext, asset
+from google.cloud import storage
+from orchestrator.logging import sentry
+
+GROUP_NAME = "connector_metrics"
+
+
+@sentry_sdk.trace
+def _safe_read_gcs_file(gcs_blob: storage.Blob) -> Optional[str]:
+    """Read the connector metrics jsonl blob.
+
+    Args:
+        gcs_blob (storage.Blob): The blob.
+
+    Returns:
+        dict: The metrics.
+    """
+    if not gcs_blob.exists():
+        return None
```

Comment:
Shall we log something here for debugging purposes?

Reply:
Yes we should!